src/main/java/Main.java

```diff
@@ -0,0 +1,81 @@
+import model.Order;
+import model.Product;
+import repo.OrderRepo;
+import repo.ProductRepo;
+import service.ShopService;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        // main menu
+        mainmenu();
+
+        Scanner scanner = new Scanner(System.in);
+
+        ArrayList<String> validInputs = new ArrayList<>();
+        validInputs.add("1");
+        validInputs.add("2");
+        String input = scanner.nextLine();
+        if(!validInputs.contains(input)){
+            System.out.println("invalid input!");
+        } else{
+            switch(input){
+                case "1":
+                    ProductRepo productRepo = new ProductRepo();
+                    ShopService shopService = new ShopService(productRepo);
+                    String answer= null;
+                    if(shopService.listProducts().isEmpty()){
+
+                        do{
```

Comment:
Mini-Tipp:
Im Prinzip stellst du dem User 3-4 Fragen.
Kleine Leerzeilen machen aus einem RIESEN Block mehrere kleine Blöcke

Viel besser oder? (Da kann man richtig aufatmen :relieved:)
```
println
scanner.next()

println
scanner.next()

println
scanner.next()
```
___
Pro-Tipp:
Du könntest den do-Block (Z- 34-40) und do-Block(Z- 45-60) jeweils in eine separate Methode auslagern.
Der Vorteil - das Programm wird auf kleine Stückchen verteilt - lesen und verstehen wird leichter

Das würde dann zB etwa so aussehen:

```java
do {
  boolean answer = scanProducts();
}
do {
  boolean answer = scanOrders(); // Der Name könnten ggf. besser sein als mein Beispiel
}
```